packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -23,70 +22,119 @@ import { IOptimismPortal2 } from "interfaces/L1/IOptimismPortal2.sol";
+    constructor(uint256 _disputeGameFinalityDelaySeconds) {
+        DISPUTE_GAME_FINALITY_DELAY_SECONDS = _disputeGameFinalityDelaySeconds;
         _disableInitializers();
     }
 
     /// @notice Initializes the contract.
     /// @param _superchainConfig The address of the SuperchainConfig contract.
     /// @param _disputeGameFactory The address of the DisputeGameFactory contract.
-    /// @param _portal The address of the OptimismPortal contract.
     /// @param _startingAnchorRoot The starting anchor root.
     function initialize(
         ISuperchainConfig _superchainConfig,
         IDisputeGameFactory _disputeGameFactory,
-        IOptimismPortal2 _portal,
-        OutputRoot memory _startingAnchorRoot
+        OutputRoot memory _startingAnchorRoot,
+        GameType _startingRespectedGameType
     )
         external
         initializer
     {
         superchainConfig = _superchainConfig;
         disputeGameFactory = _disputeGameFactory;
-        portal = _portal;
         startingAnchorRoot = _startingAnchorRoot;
+        respectedGameType = _startingRespectedGameType;
+        retirementTimestamp = uint64(block.timestamp);
+    }
+
+    /// @notice Returns whether the contract is paused.
+    /// @return Whether the contract is paused.
+    function paused() public view returns (bool) {
+        return superchainConfig.paused();
     }
 
-    /// @notice Returns the respected game type.
-    /// @return The respected game type.
-    function respectedGameType() public view returns (GameType) {
-        return portal.respectedGameType();
+    /// @notice Returns the dispute game finality delay in seconds.
+    /// @return The dispute game finality delay in seconds.
+    function disputeGameFinalityDelaySeconds() external view returns (uint256) {
+        return DISPUTE_GAME_FINALITY_DELAY_SECONDS;
+    }
+
+    /// @notice Allows the Guardian to set the respected game type.
+    /// @param _gameType The new respected game type.
+    function setRespectedGameType(GameType _gameType) external {
+        if (msg.sender != superchainConfig.guardian()) revert AnchorStateRegistry_Unauthorized();
+        respectedGameType = _gameType;
+        emit RespectedGameTypeSet(_gameType);
+    }
+
+    /// @notice Allows the Guardian to update the retirement timestamp.
+    function updateRetirementTimestamp() external {
+        if (msg.sender != superchainConfig.guardian()) revert AnchorStateRegistry_Unauthorized();
+        retirementTimestamp = uint64(block.timestamp);
+        emit RetirementTimestampSet(block.timestamp);
+    }
+
+    /// @notice Allows the Guardian to blacklist a dispute game.
+    /// @param _disputeGame Dispute game to blacklist.
+    function blacklistDisputeGame(IDisputeGame _disputeGame) external {
+        if (msg.sender != superchainConfig.guardian()) revert AnchorStateRegistry_Unauthorized();
+        disputeGameBlacklist[_disputeGame] = true;
+        emit DisputeGameBlacklisted(_disputeGame);
```

Comment:
Do we still consider blacklisting to be useful? My sense is that there's agreement that if one game is broken, all games are broken, and we'll just need to retire them or fallback to permissioned.

Reply:
Hmm, the more code we can remove the simpler things are to reason about + smaller surface area for bugs. Can we delete this in a future PR if we don't have a use case for it? Perhaps the main question to answer is are there any kinds of bugs where we can't resolve them without blacklisting?